### PR TITLE
Add known bug when upgrading to 1.17 regarding a conflict with cri-o

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>

fixes https://github.com/SUSE/avant-garde/issues/1473

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
@davidcassany can you review the content? Thanks.

**Review by davidcassany — APPROVED** [review]
Looks good to me. :+1:

If you encounter pattern related conflicts with `skuba-update` you need to upgrade to the latest k8s available and the conflicts will be gone.

**Comment by nkoranova** [comment]
> If you encounter pattern related conflicts with `skuba-update` you need to upgrade to the latest k8s available and the conflicts will be gone.

@davidcassany should we add that sentence ^ at the end of this section to make sure people know what to do? 

> `skuba-update` Error: `patterns-caasp-Node` Conflicts with CRI-O Update

File: adoc/MAIN.release-notes.adoc [hunk 1]
```diff
@@ -57,36 +57,44 @@ skuba should disable firewalld when bootstrapping/joining a node. Adds a startup
...
 
-FILLIN
+If your cluster is not up-to-date, meaning it is not in the latest {kube} version, `skuba-update` will try to install the latest version of {crio}, which will create a conflict with
+the currently installed {kube} packages.
+
+More precisely, you might encounter an error similar to this:
+
+"patterns-caasp-Node conflicts with {crio}"
+
+In that case, the recommended solution is to upgrade the cluster by running `skuba node upgrade` on each node.
```
Comment (davidcassany) [inline adoc/MAIN.release-notes.adoc]: what about:

"In that case, the recommended solution is to upgrade the cluster to the latest Kubernetes version available, this can be done by running the regular Kubernetes upgrade procedure of CaaSP [link to kubernetes upgrade docs] based on the command `skuba node upgrade`"

**Review by davidcassany — COMMENTED** [review]
> > If you encounter pattern related conflicts with `skuba-update` you need to upgrade to the latest k8s available and the conflicts will be gone.
 
 
To upgrading to the latest k8s it actually means running `skuba node upgrade apply` for each node. I considered it already covered in the current PR by 

> In that case, the recommended solution is to upgrade the cluster by running `skuba node upgrade` on each node.

Probably we could rephrase it to make more explicit.

**Comment by jordimassaguerpla** [comment]
qa gave thumbsup. We can proceed with this.

File: adoc/MAIN.release-notes.adoc (hunk 1 above)
Comment (nkoranova) [reply adoc/MAIN.release-notes.adoc]: Ah thanks, yes, that makes it better understandable I think.
